### PR TITLE
fix: Reorder versions of the same kind to prioritize k8s versions

### PR DESCRIPTION
**Pull request by ashokponkumar**
Signed-off-by: Ashok Pon Kumar <ashokponkumar@gmail.com>

### CODE REVIEW

File: internal/metadata/clusters/ibm-openshift.yaml [hunk 1]
```diff
@@ -213,38 +213,38 @@ spec:
...
       - authorization.openshift.io/v1
     Route:
       - route.openshift.io/v1
     Scale:
-      - apps.openshift.io/v1
       - apps/v1
       - apps/v1beta1
       - apps/v1beta2
       - extensions/v1beta1
       - v1
```
Comment (HarikrishnanBalagopal) [inline internal/metadata/clusters/ibm-openshift.yaml:232]: shouldn't v1 be at the top?

**Review by HarikrishnanBalagopal — CHANGES_REQUESTED** [review]
Please fix https://github.com/konveyor/move2kube/blob/main/internal/metadata/clusters/gcp-gke.yaml#L72-L80

File: internal/metadata/clusters/openshift.yaml [hunk 2]
```diff
@@ -194,24 +194,24 @@ spec:
...
       - authorization.openshift.io/v1
     Route:
       - route.openshift.io/v1
     Scale:
-      - apps.openshift.io/v1
       - apps/v1
       - apps/v1beta1
       - apps/v1beta2
       - extensions/v1beta1
       - v1
```
[the remark this answers is not in the document]
Comment (ashokponkumar) [reply internal/metadata/clusters/openshift.yaml:213]: No. That v1 is from before groups concept was started.

File: internal/metadata/clusters/ibm-openshift.yaml (hunk 1 above)
Comment (ashokponkumar) [reply internal/metadata/clusters/ibm-openshift.yaml:232]: No. That v1 is from before groups concept was started.

**Review by HarikrishnanBalagopal — APPROVED** [review]

**Pull request merged**